lms/static/js/student_account/views/RegisterView.js

```diff
@@ -127,6 +127,50 @@
                         jsHook: this.authWarningJsHook,
                         message: fullMsg
                     });
+                },
+
+                getFormData: function() {
+                    var obj = FormView.prototype.getFormData.apply(this, arguments),
+                        $form = this.$form,
+                        elements = $form[0].elements,
+                        i,
+                        len = elements.length,
+                        $el,
+                        $label,
+                        key = '',
+                        $email_element,
+                        $confirm_email_element,
+                        email = '',
+                        confirm_email = '';
+
+                    for (i = 0; i < len; i++) {
```

Comment:
I'd skip the looping, and use something like `$form.find("input[name=email]")` and `$form.find("input[name=confirm_email]")`. Those'll get you directly to the elements you need without any nasty looping.

I'm assuming here that `this.$form` is a jquery object already; otherwise you'll just need to wrap it first- `$form = $(this.$form)`.